src/data/consts.js

```diff
@@ -10,4 +11,5 @@ export const sessionTwo = {
   calendarLink: "#",
 };
 
-export const eventIcon = logo
\ No newline at end of file
+export const eventIcon1 = logo
```

Comment:
Can we rename these variables to be more descriptive?

```suggestion
export const darkThemeIcon = logo
```